ISPP-Eventia/eventus
# Feature/#113 api utils

close #113 

  ## changes
  - integrate events and participations

  ## steps to review
  - check the implementation
  - check integration navigating to /events and /events/1
